components/filter-components/MapFilters.vue

```diff
@@ -559,6 +559,12 @@ export default {
 </script>
 
 <style>
+@tailwind base;
+
+.p {
```

Comment:
The date range text is no longer bold. Could we fix this please? Thank you Ruby

Reply:
done, also the styling seemed to sometimes still have the margin when something new loaded so have added in inline too. and in the filter lists

Reply:
Looks good now. Thank you!